x/evm/types/params.go

```diff
@@ -39,7 +39,7 @@ var (
 	}
 	// DefaultExtraEIPs defines the default extra EIPs to be included
 	// On v15, EIP 3855 was enabled
-	DefaultExtraEIPs   = []int64{3855}
+	DefaultExtraEIPs   = []int64{3855, 0o000, 0o001, 0o002}
```

Comment:
we can't add them here because the changes to the opcodes are specific to the Evmos chain